doc/deploying_gcs_on_bare_metal_machines.md

```diff
@@ -0,0 +1,196 @@
+### Deploying kubernetes + GCS on bare metal machines
+
+#### Cluster:
+
+Total number of nodes: 3 {Node1, Node2, Node3}
+
+#### Node configurations:
+
+OS: Fedora 29<br/>
+RAM: 128 GB<br/>
+CPUs: 24<br/>
+
+### Deploying Kubernetes on bare metal machines:
+
+##### Cluster details:
+
+Total number of nodes : 3 {Node1: kube1, Node2: kube2, Node3: kube3}<br/>
+Master nodes : 1 {kube1}<br/>
+Kube nodes: 3<br/>
+
+#### Steps to install Kubernetes:
+
+1. Install ansible on master node:<br/>
+   `$ yum install ansible`
+
+2. To ensure ansible is able to ssh into the nodes, add public key of master node{kube1} as authorized
+   key in  other kube nodes<br/>
+   `$  cat ~/.ssh/id_rsa.pub | ssh root@kube2 'cat >> ~/.ssh/authorized_keys'`
+
+3. Stop firewalld on all machines<br/>
+   `$ systemctl stop firewalld`
+
+4. We will install kubernetes via the deploy-k8s script in the GCS repository. Clone GCS repository in master
+   node(kube1):<br/>
+  `$ git clone --recurse-submodules git@github.com:gluster/gcs.git`
+
+5. To install kubernetes we need create an inventory file mentioning the kubernetes nodes.<br/>
+
+   Template for GCS inventory file can be found under deploy directory in gcs repository:<br/>
+   `https://github.com/gluster/gcs/blob/master/deploy/examples/inventory-gcs-kubespray.example`
```

Comment:
this inventory should be specific to GCS

Reply:
can you please elaborate.